src/aatoolbox/utils/geoboundingbox.py

```diff
@@ -89,6 +89,7 @@ def round_boundingbox_coords(
             elif direction in ("south", "west"):
                 function = np.floor.__call__  # needed for mypy
                 offset_factor = -1
+
```

Comment:
arghh the `rounded_coord` has to deal with floating inprecision. So we get outputs like `32.80000000000000182076576039` while we would like just `32.8`. This is problematic cause when `round_coords()` is then called twice, it rounds up again which we wouldn't want. 

@turnerm Do you know a solution for this? The internet seems to suggest using `Decimal()` but wondering if you have a better idea?

Reply:
Ah yes I think using decimal would make a lot of sense here. Furthermore, it might be good to make it only possible to run `round_coords` once (how is it getting run twice?). But this could be moved to an issue.

Reply:
👍 it is being run twice due to the lovely combination class but even then I think it would be good to have a check that it isn't accidentally ran twice in some way. Do you know how we could do that?

Reply:
You could make an instance attributed called `_is_rounded` that is set to `True` once the coords have been rounded, and then when running `round_coords` just check this attribute and exit out if it's `True`